api.py

```
@@ -196,7 +196,7 @@ def set_bot(bot) -> None:
         db_manager._pool = None
 
 
-def _get_pool():
+def _get_pool() -> Pool | None:
```

Flake8 (7.3.0)

[error] undefined name 'Pool'

(F821)